info/v2/conversions.go

```diff
@@ -0,0 +1,319 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
(nit) can you rename this file to conversion.go so that git picks up the diff correctly?